sites/docs/src/content/docs/usage/configuration/system_requirements.md

```diff
@@ -0,0 +1,121 @@
+See the [Nextflow executor documentation](https://www.nextflow.io/docs/latest/executor.html#executor-page) for more information about specific schedulers.
+
+### Max resources
+
+Pipeline runs occasionally fail when a process requests more resources than you have available on your system.
+
+To avoid these failures, all nf-core pipelines [check](https://github.com/nf-core/tools/blob/99961bedab1518f592668727a4d692c4ddf3c336/nf_core/pipeline-template/nextflow.config#L206-L237) pipeline-step resource requests against parameters called `--max_cpus`, `--max_memory` and `--max_time`. These parameters can be set as the maximum possible resources of a machine or node and act as a cap that prevents Nextflow submitting a single job requesting resources more than what is possible.
+
+:::warning
+Increasing these values from the defaults will not increase the resources available to the pipeline tasks! See [Tuning resources](#tuning-resources) for this.
+:::
+
+Most pipelines will attempt to automatically restart jobs that fail due to lack of resources with double-requests. These `--max_<resource>` caps keep requests from getting out of hand and crashing the entire pipeline run. If a particular job exceeds the process-specific default resources and is retried, only resource requests (cpu, memory, or time) that have not yet reached the value set with `--max_<resource>` will be increased during the retry.
+
+:::warning
+The `--max_<resource>` parameters do not represent the total sum of resource usage of the pipeline at a given time - only a single pipeline job!
+:::
+
+### Tuning resources
+
+The base config of nf-core pipelines defines the default resources allocated to each different step in the workflow (e.g., in a [`base.config`](https://github.com/nf-core/rnaseq/blob/master/conf/base.config) file).
+
+These values are deliberately generous due to the wide variety of workloads done by different users.
+As a result, you may find that the jobs are given more resources than they need and your system is not used efficiently.
+At the other end of the scale, you may want to increase the resources given to a specific task to make it run faster.
+You may wish to increase resources if you get a pipeline reporting a step failing with an `Command exit status`, such as `137`.
+
+Where possible, tools are tuned to make use of the resources available, for example with a tool parameter (e.g., `-p ${task.cpus}`, where `${task.cpus}` is dynamically set according to what has been specified in the pipeline configuration files). However, this is not possible with all tools, in which case required resources are estimated for an average user.
+
+Workflow resources can modified through [shared configuration profiles](#shared-configuration-files) or [custom configuration files](#custom-parameter-and-configuration-files) to match your requirements.
```

Comment:
```suggestion
```

Again this is referring to sections not presetn, I guess we coiuld refer back to the previous page?